toolkit/src/components/cut/list-item/index.hbs

```diff
@@ -0,0 +1,32 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<div ...attributes>
+  {{#if (or this.href this.route)}}
+    <Hds::Interactive
```

Comment:
one thing I noticed, but I am not sure you can use (depends on how the `onClick` at line 20 is defined): the `Hds::Interactive` automatically generates a `<button>` element if no `href` or `route` is provided ([see here](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/interactive/index.hbs))

Reply:
yea, that's true. 

Hds::Interactive is in block: 
```{{#if (or this.href this.route) }}{{/if}}```

So for our case we need it only to be a link

Reply:
Update: Tyler shared the idea, that we actually can use Hds::Interactive with <button> scenario as well, so I changed a bit the code and now it shows interactive in case, when we have` @onClick, @href or @route` passed